platform/chronos/Dockerfile

```diff
@@ -1,34 +1,56 @@
-ARG RUST_VERSION=1.80.1
-ARG APP_NAME=discord
+FROM lukemathwalker/cargo-chef:latest-rust-1.59.0 AS chef
```

Comment:
why such an old version of rust or is this the version of the chef?

Reply:
I don't actually know. It was broadly a copy-paste job from the `chronos` service, @chrisaddy.